src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -1727,10 +1727,12 @@ public static void loadGameCharacterVariablesFromXML(GameCharacter character, St
 		if(playerSlaveryElement!=null) {
 			element = (Element) playerSlaveryElement.getElementsByTagName("rules").item(0);
 			
-			if(element.getElementsByTagName("rule").getLength() > 0)
+			NodeList ruleElements = element.getElementsByTagName("rule");
+			
+			if(ruleElements.getLength() > 0)
```

Comment:
This check can go; it's a duplicate from the for loop. It will automatically get checked with ``i=0; i < ruleElements.getLength()``